ObsoleteModelica4.mo

```diff
@@ -2875,16 +2875,16 @@ Obsolete saliency cage model, see
       end BasicMachines;
     end FundamentalWave;
   end Magnetic;
-  annotation (uses(Modelica(version="4.0.0")),
-              version="4.0.0",
-              versionDate="2020-06-04",
-              dateModified = "2020-06-04 11:00:00Z",
+  annotation (uses(Modelica(version="4.1.0")),
+              version="4.1.0",
+              versionDate="2024-03-15",
+              dateModified = "2024-01-01 11:00:00Z",
               revisionId="$Format:%h %ci$",
 Documentation(info="<html>
 <p>
 This package contains models and blocks from the Modelica Standard Library
 version 3.2.3 that are no longer available in version 4.0.0
-The conversion script for version 4.0.0 changes references in existing
+The conversion script for versions >= 4.0.0 changes references in existing
```

Comment:
Is the next time we will have conversion scripts going to be 5.0.0 (according to semantic versioning)? I think so.

Will we at that point introduce an `ObsoleteModelica5.mo`? Probably?

I think this change is unnecessary and confusing. Same for the one on line 2900 below.